packages/logger-prettify/src/index.ts

```diff
@@ -1,18 +1,20 @@
-import { printMessage } from './formatter';
+import { fillInMsgTemplate, printMessage } from './formatter';
 import { PrettyOptionsExtended } from './types';
 
+export { fillInMsgTemplate };
+
 export type PrettyFactory = (param) => string;
 
 /*
  options eg:
  { messageKey: 'msg', levelFirst: true, prettyStamp: false }
  */
 
-module.exports = function prettyFactory(options: PrettyOptionsExtended): PrettyFactory {
+export default function prettyFactory(options: PrettyOptionsExtended): PrettyFactory {
```

Comment:
I have not seen anything but problems with default imports. Is this necessary or part of some new Verdaccio guidelines?

Reply:
> I have not seen anything but problems with default imports.
Could you share some? I'm curious. No strict guidelines on this, as long is ES6 syntax we are fine.

Reply:
Just for context, Pino expect `default`, eg: https://github.com/pinojs/pino-pretty/blob/master/index.js#L217

Reply:
@Splaktar - it was technically the default export prior but used the commonJS syntax. I was aiming to try to change/refactor as little code as possible with this PR. Doing a _rough_ search within the Verdaccio source code I see close to 200 other uses of `export default ...` so it aligns with the current coding style/conventions used within Verdaccio.

Reply:
It's probably fine in NodeJS apps, most issues I've seen with them are in web apps due to webpack and bundle splitting, etc.